src/telcameras_v2/tools.py

```diff
@@ -90,3 +93,8 @@ def scramble_count_aggregate(count_aggregate):
             count_aggregate.count_scrambled = count_aggregate.count + randint(-1, 1)
 
     return count_aggregate
+
+
+def get_messages_count_for_past_minutes(model_class, timestamp_fieldname, minutes):
+    check_from = now() - timedelta(minutes=minutes)
+    return model_class.objects.filter(**{f'{timestamp_fieldname}__gte': check_from}).count()
```

Comment:
Dit soort code hoort eigenlijk in de QuerySet class of de manager class van een model. Dan krijg je:

`model_class.objects.last_minutes(minutes=x)` (in de QuerySet class) of zelfs `model_class.last_minutes(minutes=x)` (op de manager).